tools/cli/src/build.rs

```diff
@@ -59,12 +62,16 @@ pub(crate) fn build(trailing_args: Vec<&str>) -> CLIResult<()> {
                 .filter(|name| name.ends_with(".wasm"))
                 .collect::<Vec<_>>();
             if !new_wasms.is_empty() {
-                if let Ok(sdk_version) = extract_sdk_version(&manifest_path) {
-                    wasms.extend(
+                match extract_sdk_version(&manifest_path) {
+                    Ok(sdk_version) => wasms.extend(
                         new_wasms
                             .into_iter()
                             .map(|name| (name, sdk_version.clone())),
-                    )
+                    ),
+                    Err(ManifestError::NonMarineWasm) => {
+                        log::info!("Skipping {} as non-marine wasm", manifest_path.display())
+                    }
+                    Err(e) => log::error!("marine-rs-sdk version extraction failed: {}", e),
```

Comment:
Shouldn't we propagate the error?

Reply:
I decided to keep silently ignoring all errors, because @mikevoronov suggests to be careful with adding messages for user.

Reply:
I this that this one worth to be shown to an user.

Reply:
As I added env_logger to the build command, this error is shown to the user by default, like this:
```[2023-03-02T14:50:47Z ERROR marine::build] marine-rs-sdk version extraction failed: <ManifestError::display()>```, but it does not stop execution